.github/PULL_REQUEST_TEMPLATE/bug_fix.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Fix/Demonstration
+about: This fixes or demonstrates a bug in SQLFluff
+labels: bug
+---
+
+-- Firstly, thanks for fixing a bug! Secondly, please make sure your PR includes a test case demonstrating that the bug is fixed. Test cases should be as minimal as possible to show it's fixed but also be representative of the case the original reporter of the bug posted. If this PR is just a failing test case, please file it as a `draft` PR or use pytest `xfail` to mark the case as expected to fail. --

```

Comment:
Could it be helpful to document the more common types of test cases and how to create them? I'm mainly thinking of the standalone `.sql`/`.yaml` parser test cases and the `.yml` lint rule test cases, but there may be others.